### ISSUE TITLE
Is this project still actively maintained, or is it abandoned?

### ISSUE THREAD

**Issue by 0xdevalias** [jekyll/classifier-reborn#193]
As I note in https://github.com/jekyll/classifier-reborn/issues/192, it seems that some of the recommended library choices for speeding this project up are quite outdated, and themselves seem unmaintained, or have been replaced by newer versions.

I see that the latest commit to this project was in February 2020, yet the last released version was in December 2017, with [at least one issue](https://github.com/jekyll/classifier-reborn/issues/182) noting that more recent versions haven't been released to rubygems.

A lot of the open issues seem quite old as well.

It would be great to get some insight from project maintainers about the status of this project.

**Comment by mkasberg** [comment]
To add some additional context to the above - I have a Jekyll site with ~50 posts. I typically build this site on Ubuntu Linux using `jekyll build --lsi`.

On Ubuntu 16.04, with `libgsl-dev` installed, my site builds in under 2 seconds. (Ubuntu 16.04, Ruby 2.6, gsl 2.1)

On Ubuntu 20.10, with `libgsl-dev` installed, the identical jekyll project takes 7m51s to build on the same hardware. (Ubuntu 20.10, Ruby 2.7, gsl 2.6)

It seems clear to me that LSI with native GSL is broken on recent versions of Ubuntu (at least, without building and installing older versions of GSL). It's unclear to me whether this is a problem with classifier-reborn or with rb-gsl (which also appears to have little recent activity). It's also unclear whether this is just a simple problem with dependency versions or if it will require more work to fix. I plan on investigating more myself as I have time, though it would be great if anyone who might know more about LSI and GSL than me could point me in the right direction.

**Comment by mkasberg** [comment]
After more testing, I was able to get fast results in a Docker container with Ubuntu 20.10 and GSL 2.6. It's unclear to me why my host machine was building slowly when not using a Docker container, but reinstalling some gems seemed to fix it.

I did `gem uninstall gsl` and `gem uninstall classifier-reborn`, followed by `bundle install`. After reinstalling those gems, LSI was very fast again! Sorry for the extra chatter in this thread, but hopefully this information is useful.
